packages/swingset-runner/.gitignore

```diff
@@ -1,4 +1,5 @@
 # swingset persistence data
+swingset-kernel-state
```

Comment:
maybe remove this?

Reply:
`.gitignore` already ignores `.mdb` files, but the presence of transcript files make git think the store directories need to be added.  I suppose we could add a filename extension to the transcript files (which, I suppose, you might make a case we should do anyway) and tell git to ignore **those**, but _something_ will have to go in `.gitignore` or I end up with an 50 extra lines of crap in the output every time I do a `git status`.

Reply:
All the unit tests either use the in-memory version of the store or clean up after themselves, but swingset-runner demos leave their databases around for inspection or replay.  I think I'm leaning towards giving the stream files their own file name extension so they can be ignored specifically.  That way if any _other_ stuff gets put (perhaps inadvertently) in a `swingset-kernel-state` directory it will be flagged by `git status`.